code/xcvm/lib/core/src/transport/ibc/ics20/pfm.rs

```diff
@@ -15,7 +15,7 @@ pub struct ForwardingMemo {
 	#[serde(skip_serializing_if = "Option::is_none")]
 	pub channel: Option<ChannelId>,
 	#[serde(skip_serializing_if = "Option::is_none")]
-	pub timeout: Option<String>,
+	pub timeout: Option<u64>,
```

Comment:
Note that serde uses integer representation for u64.  Serde talking to serde will work fine but we may run into problems if this memo would be ever read by JavaScript for example.  Perhaps better use `Displayed<u64>`.

Reply:
pub height: Option<u64>,
works fine so i just replaced String with u64 for other field.